source/options.css

```diff
@@ -31,13 +22,13 @@ h3,
 h4 {
 	width: 100%;
```

Comment:
`width: 100%` is essentially the default for block elements.

Reply:
We have some `h4` elements inside `label` elements, that have `display: flex` applied. This is to make other content wrap.